src/center.jl

```diff
@@ -1,6 +1,35 @@
 export hchebyshevcenter, vchebyshevcenter, chebyshevcenter
 using JuMP
 
+"""
+    maximum_radius_with_center(h::HRep, center)
+
+Return the maximum radius `r` such that the Euclidean ball of center `center`
+and radius `r` is included in the polyhedron `h`.
+"""
+function maximum_radius_with_center(h::HRep{T}, center) where T
+    if any(hp -> !isapproxzero(coord(hp)), hyperplanes(h))
+        return zero(T)
+    else
+        radius = nothing
+        for hs in halfspaces(h)
+            n = norm(hs.a, 2)
+            if iszero(n)
+                if hs.β < 0
+                    # The polyhedron is empty
+                    return zero(T)
+                end
+            else
+                radius = min(radius, (hs.β - center ⋅ hs.a) / n)
```

Comment:
I think this won't work at the first iteration because:
```julia
julia> min(nothing, 2.5)
ERROR: MethodError: no method matching isless(::Float64, ::Nothing)
Closest candidates are:
  isless(::Missing, ::Any) at missing.jl:87
  isless(::Float64, ::Float64) at float.jl:465
  isless(::AbstractFloat, ::AbstractFloat) at operators.jl:165
  ...
```

Shall we add a check above? Something like `isnothing(nothing) && (radius = (hs.β - center ⋅ hs.a) / n)`?

Reply:
Actually, adding `isnothing(nothing) && (radius = (hs.β - center ⋅ hs.a) / n)` is not ideal because `radius = min(radius, (hs.β - center ⋅ hs.a) / n)` would run afterwards unnecessarily. Adding another branch to the `if` statement may be better.